db/migrations/20230706150233_endpoints_evals.sql

```diff
@@ -0,0 +1,30 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE unweave.eval (
+    id text NOT NULL PRIMARY KEY,
+    exec_id text NOT NULL,
+    project_id text NOT NULL,
```

Comment:
Shouldn't these have foreign key references to `unweave.exec` and `unweave.project`?